### DISCUSSION TITLE
Representation of Specimen in Laboratory Tests in Modules

### DISCUSSION THREAD

**Discussion by paulolaup** [synthetichealth/synthea#1363] [General]
Hi,
I'm currently working on evaluating different FHIR servers regarding their suitability for feasibility query answering and I'm planing to use Synthea to generate a sufficiently large test data set on which to run these queries.

I want to base the test queries on inclusion and exclusion criteria of real studies, which sometimes (especially in oncology) link the presence of a condition to test performed on some specimen.

While running Synthea, I noticed the lack of such resource instance data.

Are there ways to represent such data in Synthea or can one extract it from the generated data?

**Comment by jawalonoski** [comment]
If you don't see, it probably isn't there.

You can double-check what might appear in the records, by browsing the module builder: https://synthetichealth.github.io/module-builder/

For example, if you are looking at breast cancer, you'd select the breast cancer module: https://synthetichealth.github.io/module-builder/#breast_cancer

Please be aware that there are submodules which might contain additional clinical content (basically these submodules are like functions or subroutines), which you can find by clicking on `"Open Module" > "Submodules" > [Submodule]`

You can add missing diagnostic tests using the point-and-click interface and adding the appropriate terminology code, downloading the module with your changes (JSON file), and using that module in Synthea. See https://github.com/synthetichealth/synthea/wiki/Generic-Module-Framework%3A-Running-New-Modules
